gplog/gplog.go

```diff
@@ -72,14 +72,16 @@ const (
  *          cannot be reached.  This function will exit the program after printing
  *          the error message.
  */
+type LogPrefixFunc func(string) string
```

Comment:
Since this is only used in one place in the struct definition, I don't think we need to declare a type for this.

Reply:
I guess it can't hurt for library users to be able to be more explicit with their function typing.